packages/@misk/core/src/features/Navbar/Component.tsx

```diff
@@ -66,7 +66,8 @@ export class Navbar extends React.Component<INavbarProps, {}> {
       menuShowButton,
       navbar_items,
       status,
-      theme = defaultTheme
+      theme = defaultTheme,
+      items_alignment,
```

Comment:
I'm leaning towards this field being named `navbar_items_alignment` so that it's clear that the alignment is associated with the `navbar_items` group